internal/service/privatelinkendpointserviceserverless/resource_privatelink_endpoint_service_serverless_migration_test.go

```diff
@@ -38,3 +39,32 @@ func TestMigServerlessPrivateLinkEndpointService_basic(t *testing.T) {
 		},
 	})
 }
+
+func TestMigServerlessPrivateLinkEndpointService_noComment(t *testing.T) {
```

Comment:
[q] not sure from the title and steps of this migration test what scenario it is testing. I see it defines a "test comment" on both steps (no update), so I would assume since no update is made this was also working before?